script/tidy

```diff
@@ -5,16 +5,31 @@
 #
 
 check=
+assume_already_tidy=
 if test "$1"  = '--check'; then
     shift
     check=1
 fi
+assume_already_tidy=
+if test "$1"  = '--assume-already-tidy'; then
+    shift
+    assume_already_tidy=1
+fi
 
 if ! which perltidy > /dev/null 2>&1; then
     echo "No perltidy found, install it first!"
     exit 1
 fi
 
+function find_changed_files() {
+    changed_files=()
+    for file in "${files[@]}"; do
+        if ! [[ $assume_already_tidy ]] || [ "$(git diff $file)" ]; then
```

Comment:
I'd think that git diff --name-only makes it even faster as there is no diff to be calculated